plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -256,6 +256,8 @@
     public static final String ORC_BLOOM_FILTER_COLUMNS_KEY = "orc.bloom.filter.columns";
     public static final String ORC_BLOOM_FILTER_FPP_KEY = "orc.bloom.filter.fpp";
 
+    private static final FunctionName NUMBER_OF_DISTINCT_VALUES = new FunctionName("approx_distinct");
```

Comment:
nit: change the name of the constant so that it doesn't collide with `ColumnStatisticType.NUMBER_OF_DISTINCT_VALUES`

Reply:
it's intentionally